SCClassLibrary/Common/Control/Server.sc

```diff
@@ -339,6 +340,8 @@ Server {
 		this.name = argName;
 		all.add(this);
 
+		pidReleaseCondition = Condition({ this.pid == nil });
```

Comment:
Like ```pid```, ```pidReleaseCondition``` is an instance variable, because quitting is individual/independent per server instance. Not sure I understand your question?

Reply:
but you are setting it in `*initClass`, here.

Reply:
… I've just read the original code, false alarm, it is all in `init`. This is a github/diff artefact.

Reply:
OK :-)

Reply:
will go over this anyway in the ServerProcess refactor/cleanup, where every ServerProcess state gets its own condition, so this is just to make all tests with supernova to run now, rather than waiting for the  ServerProcess refactor/cleanup to get through.